install/kubernetes/hubble/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: hubble
-version: 0.0.1
+version: 0.0.2
```

Comment:
We haven't been very diligent about actually nudging this in the past, thanks for pointing this out.